README.md

```diff
@@ -85,17 +77,23 @@ Legend:
 ✅ quay.io/prometheus/prometheus:v2.42.0
 ✅ redis:6.2.4-alpine
 ✅ registry.k8s.io/autoscaling/cluster-autoscaler:v1.25.3
-🚫 registry.k8s.io/ingress-nginx/controller:v1.9.4@sha256:5b161f051d017e55d358435f295f5e9a297e66158f136321d9b04520ec6c48a3
 ✅ registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.8.1
 ```
 
 ### Errors in Output
 
-At present the tool will show an error if it cannot connect to the repository for an image for any reason.
-This could include:
-* Authentication failure
-* Communication error
-* Image no longer available
+If there is an error whilst checking an image, the tool will display the 🚫 symbol and give a short description of the error at the end of the line. The current common errors are:
+
+- Authentication error.
+- Communication error.
+- Image not found. Some pods like `example-pod` are using an image that no longer exists.
+- Unknown error, run in debug mode using the flag `-d` for more info
+
+Authentication errors usually mean something went wrong with docker credentials.
+Communication errors usually mean that the client cant reach the repository host
+Image not found errors happen when the image that is running in the cluster no longer exists in the registry.
+
+A more detailed error can be found when running the tool with the flag `-d`. It also shows you all the pods that use this image
```

Comment:
Great to have the extra information, but the description should be actionable, also it would be good to combine this in a the bullet point or in a table. For example:

- `Authentication error` - a private image could not be be checked, check the docker credentials are present and up to date.

I'm not saying my suggestion is correct, but you can hopefully see how it explains the reason and gives a possible resolution.